### DISCUSSION TITLE
How to use

### DISCUSSION THREAD

**Discussion by graid2030** [abrahamjuliot/creepjs#218] [Q&A]
Hi! Can I use creepjs to find bots on my website? Do you have some guidance on how to do it? Thank you!

**Comment by abrahamjuliot** [comment]
I don't recommend it. The design of this project is intended for research and educational insight. We want to avoid creating a fingerprinting library used for real-world web traffic.

**Comment by graid2030** [comment]
Unfortunately, I haven't found many solutions for this. For example, https://github.com/fingerprintjs/BotD is far inferior to creepjs, and its paid counterpart is too expensive

**Comment by Sinjarulsk** [comment]
> Hi! Can I use creepjs to find bots on my website? Do you have some guidance on how to do it? Thank you!

Can you help me please I won't create chrome browser Automation
